submissions/mwabaChabo/HTML/products_page.html

```diff
@@ -5,6 +5,8 @@
     </head>
 
     <body style="background-color: grey;">
+        <a href="file:///C:/Users/ACER/Desktop/Dufuna-CodeCamp-21/submissions/mwabaChabo/HTML/products_page.html">Product</a><br>
```

Comment:
This path is absolute and will only work at your end because this path doesnt exist for every other person. You should have something like this "products_page.html" for the Products page and then do "../HTML-FORMS/contact_form_page.html" for the contact page since it is in another folder. 

Repeat this similar step for Contact too